lib/routes/account.js

```diff
@@ -252,6 +252,18 @@ module.exports = function (
             tokenVerificationId = undefined
           }
 
+          if (query._createdAt) {
+            // We don't expect this to be set outside the tests so log an error
+            // if we do encounter it, to help debug what's going on.
+            log.error({
+              op: 'account.create.createSessionToken',
+              err: new Error('Unexpected _createdAt query parameter'),
+              _createdAt: query._createdAt,
```

Comment:
I wanted the properties on the error object to have the same names as the variables they're documenting, to be totally unambiguous and aid greppability. But perhaps I'm imagining benefits that don't exist? Happy to make the change if you think so?